Maple2.Model/Game/Party/PartyMember.cs

```diff
@@ -0,0 +1,90 @@
+using System;
+using System.Numerics;
+using System.Threading;
+using Maple2.Model.Common;
+using Maple2.PacketLib.Tools;
+using Maple2.Tools;
+using Maple2.Tools.Extensions;
+
+namespace Maple2.Model.Game;
+
+public class PartyMember : IByteSerializable, IDisposable {
+    public long PartyId { get; init; }
+    public required PlayerInfo Info;
+    public long JoinTime;
+    public long LoginTime;
+    public long AccountId => Info.AccountId;
+    public long CharacterId => Info.CharacterId;
+    public string Name => Info.Name;
+    public byte ReadyState = 0;
+
+    public CancellationTokenSource? TokenSource;
+
+    public void WriteTo(IByteWriter writer) {
+        writer.WriteLong(Info.AccountId);
+        writer.WriteLong(Info.CharacterId);
+        writer.WriteUnicodeString(Info.Name);
+        writer.Write(Info.Gender);
+        writer.WriteByte(1);
+        writer.WriteLong();
+        writer.WriteInt();
+        writer.WriteInt(Info.MapId);
+        writer.WriteInt(Info.MapId);
+        writer.WriteInt(Info.PlotMapId);
+        writer.WriteShort(Info.Level);
+        writer.WriteShort(Info.Channel);
+        writer.WriteInt((int) Info.Job.Code());
+        writer.Write(Info.Job);
+        writer.WriteInt((int) Info.CurrentHp);
+        writer.WriteInt((int) Info.TotalHp);
+        writer.WriteShort();
+        writer.WriteLong();
+        writer.WriteLong();
+        writer.WriteLong();
+        writer.WriteInt();
+        writer.Write(new Vector3());
```

Comment:
Here and elsewhere, for structs you can just use `default`. Also for structs always include type in `Write` call to prevent bugs where someone accidentally changes the type. (Basically there should be no instances of `writer.Write(...)`)

Example:
```cs
writer.Write<Vector3>(default);
```